templates/KnockoutSpa/tsconfig.json

```diff
@@ -3,8 +3,13 @@
     "moduleResolution": "node",
     "target": "es5",
     "sourceMap": true,
-    "skipDefaultLibCheck": true
+    "skipDefaultLibCheck": true,
+    "typeRoots": [ "node_modules/@types" ],
+    "types": ["core-js", "crossroads", "history", "knockout",
+     "requirejs", "signals", "whatwg-fetch" ]
   },
+  "compileOnSave": false,
+  "buildOnSave": false,
```

Comment:
They are just helpful depending on certain IDE's to disable automatic `tsc` compilation / etc.
Do you want me to remove them? Just thought they were helpful to ensure webpack handles everything.